narratives/ncov_sit-rep_2020-04-17.md

```diff
@@ -0,0 +1,238 @@
+* [The NYTimes COVID-19 coverage](https://www.nytimes.com/news-event/coronavirus).
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 3-->
+# [A note on sampling](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=map&f_country=USA&p=full&r=division)
+We currently have sequences from samples collected in 36 states. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
+<br><br>
+While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
+<br><br>
+For example, the map shows relatively few sequences from the Southeastern United States. This is NOT because COVID-19 isn't circulating in these areas, or that these cases are not as crucial to understand; rather, we just don't have much data available from these areas. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 4-->
+# [Main New York clade linked to European outbreak](https://nextstrain.org/ncov/north-america/2020-04-17?c=division&d=tree,map&f_division=Europe,New%20York&label=clade:A2&p=full&r=division)
+
+As you may have seen [in the news this week](https://www.nytimes.com/2020/04/08/science/new-york-coronavirus-cases-europe-genomes.html), most of the viral samples from New York (in color) group closely with samples from Europe (gray).
+<br><br>
+The New York clade is both flanked by and intercalated with sequences from Europe, which strongly suggests that most early cases resulted from introductions from Europe.
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 5-->
+# [Many introductions to New York](https://nextstrain.org/ncov/north-america/2020-04-17?label=clade:A2&c=division&d=tree,map&f_division=Europe,New%20York,Asia&p=grid&r=division)
+
+Although the majority of the New York outbreak was seeded by European introductions, by adding in more contextual sequences, we get a more nuanced view.
+True to its cosmopolitan nature, there are also cases in New York that are closely related to samples from every other region of the world.
+<br><br>
+Notably, although the early New York cases were most likely represent inbound transmission, the directionality of these transmissions is less clear later on in the outbreak. As we reported on [last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-10?n=11), there is also evidence for recent reintroductions from the U.S. to Europe and Asia.
```

Comment:
```suggestion
Notably, although the early New York cases likely represent inbound transmission, the direction of transmissions is less clear later on in the outbreak. As we reported on [last week](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-10?n=11), there is also evidence for recent reintroductions from the U.S. to Europe and Asia.
```

Reply:
For people who don't know how to read a tree, I think it's worth explaining what you're referring to here, i.e. the mixing of NY samples with samples from other locales. Could be a good place to remind readers of the uncertainty of phylogenetic tree reconstructions.